Cargo.toml

```diff
@@ -126,8 +126,8 @@ datafusion-common = { git = "https://github.com/spiceai/datafusion.git", rev = "
 datafusion-expr = { git = "https://github.com/spiceai/datafusion.git", rev = "fbedf88bb46b3b9b680977a80f76a5ee8d4a9bc4" }
 datafusion-execution = { git = "https://github.com/spiceai/datafusion.git", rev = "fbedf88bb46b3b9b680977a80f76a5ee8d4a9bc4" }
 
-datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "ce1ce6158ce03f88c67cde25fca00f436c3e1690" }
-datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "2bcf481b4abe9d0bd6bb2479ce49020df66ff97f" }
+datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "0aabd423ef8aea0ae05479331d1486e9ef8f8061" }
```

Comment:
Note: this enables filters pushdown support optimization. I'll ensure latest unparser improvements to correctly support this work are merged today so everything works as expected

Reply:
@sgrebnov Out of curiosity are the two connected? Pushdown and native schema inference?

Reply:
@slyons - `table-providers` crate where new schema inference added depends on `datafusion-federation` where we've added Pushdown optimization and also alrady included to `table-providers` so any upcoming change in `table-providers` will automatically bring updated version of `datafusion-federation`